src/logger/fileWriter.c

```diff
@@ -297,18 +253,7 @@ static void open_log_file(struct logging_status *ls)
                 return;
         }
 
-
-        fs_lock();
-        bool fs_good = sdcard_fs_mounted();
-        if (!fs_good) {
-                FRESULT initfs_rc = InitFS();
-                pr_info_int_msg(_LOG_PFX "Remounting filesystem: ", initfs_rc);
-                fs_good = (initfs_rc == FR_OK);
-                if (FR_OK != initfs_rc) {
-                        pr_error_int_msg(_LOG_PFX "Error Initializing filesystem: ", initfs_rc);
-                }
-        }
-        fs_unlock();
+        bool fs_good = sdcard_ready( false ); // true == attempt remount immediately even if just attempted.
```

Comment:
This code did not used to have a retry delay, so I have affected the implementation here. @brentpicasso Please scrutinize and suggest improvements if you have them.